setup_dev_env.sh

```diff
@@ -23,4 +27,4 @@ export RETHINK_HOST=$LH
 
 export REDIS_HOST=$LH
 
-pipenv shell
+[[ "$LH" != "192.0.0.1" ]] || pipenv shell
```

Comment:
So, why not using the pipenv shell also in Travis?

Reply:
According to my understanding "pipenv shell" command is used to install quantumleap as a source for running of app.py file. In case of travis, all the services are running as a containerized enviroment, so there is no need to run app.py file for running quantumleap service in case of travis.

Reply:
see my comment above about `pipenv shell`. I think we should run it in Travis too so we should actually rollback this line with the conditional statement?

Reply:
About ```pipenv shell```, I have investigated and found an issue on [GitHub](https://github.com/pypa/pipenv/issues/3500). Please look into this and provide your opinion on it.

Reply:
@SourabhChourasiya-NEC thanks for pointing it out!
There's something I should clear out though. My comment earlier was misleading: we actually never run this script on Travis, we only use it to set up a *local* test environment on our boxes. But I suppose we can leave this file as is, it looks like your changes actually make the script more robust, thanks!